src/reflect/scala/reflect/internal/Scopes.scala

```diff
@@ -229,36 +239,41 @@ trait Scopes extends api.Scopes { self: SymbolTable =>
     /** Lookup a symbol.
      */
     def lookup(name: Name): Symbol = {
+      val startTime = if (statistics.canEnable) statistics.startTimer(statistics.scopeLookupTime) else null
```

Comment:
I don't think this is really a suitable place to instrument, as it misses a lot of important lookups that use alternative scope lookup APIs. E.g. `Context.isQualifyingImplicit` uses `lookupEntry` and `Context.lookupSymbol` uses `lookupUnshadowedEntries`.

Reply:
Oh, I see you've instrumented `lookupEntry` as well below. Should we then start this timer below the call to `lookupEntry` to avoid double counting?

Reply:
I instrumented both so that `lookupEntry` also records the time spent in executing its whole body, which relies on `lookupNextEntry`. I think that the statistics infrastructure prevents double counting by design (nested calls do not record the times if there are other timers wrapping them) -> a fresh pair of eyes to confirm this would be good.
  
However, maybe measuring `lookupEntry` is uninteresting... so I've removed it in my last commit.